habitat-baselines/habitat_baselines/rl/hrl/hl/fixed_policy.py

```diff
@@ -23,39 +21,25 @@ class FixedHighLevelPolicy(HighLevelPolicy):
 
     _solution_actions: List[Tuple[str, List[str]]]
```

Comment:
can we rename this to be the 'PDDLHighLevelPolicy'?
Is there a reason to call this 'Fixed'?

Reply:
Yes, there is a reason. This is not a STRIPS planner. It only executes a hardcoded list of HL actions.